doc/source/rllib/rllib-algorithms.rst

```diff
@@ -7,236 +7,176 @@
-
-.. tip::
-
-    If you need to scale out with GPUs on multiple nodes, consider using `decentralized PPO <#decentralized-distributed-proximal-policy-optimization-dd-ppo>`__.
-
-.. figure:: images/ppo-arch.svg
-
-    PPO architecture
+.. _sac:
 
-Tuned examples:
-`Unity3D Soccer (multi-agent: Strikers vs Goalie) <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/ppo/unity3d-soccer-strikers-vs-goalie-ppo.yaml>`__,
-`Humanoid-v1 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/ppo/humanoid-ppo-gae.yaml>`__,
-`Hopper-v1 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/ppo/hopper-ppo.yaml>`__,
-`Pendulum-v1 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/ppo/pendulum-ppo.yaml>`__,
-`PongDeterministic-v4 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/ppo/pong-ppo.yaml>`__,
-`Walker2d-v1 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/ppo/walker2d-ppo.yaml>`__,
-`HalfCheetah-v2 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/ppo/halfcheetah-ppo.yaml>`__,
-`{BeamRider,Breakout,Qbert,SpaceInvaders}NoFrameskip-v4 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/ppo/atari-ppo.yaml>`__
+Soft Actor Critic (SAC)
+------------------------
+`[original paper] <https://arxiv.org/pdf/1801.01290>`__,
+`[follow up paper] <https://arxiv.org/pdf/1812.05905.pdf>`__,
+`[discrete actions paper] <https://arxiv.org/pdf/1910.07207v2.pdf>`__,
+`[implementation] <https://github.com/ray-project/ray/blob/master/rllib/algorithms/sac/sac.py>`__.
 
+.. figure:: images/algos/dqn-architecture.svg
 
-**Atari results**: `more details <https://github.com/ray-project/rl-experiments>`__
+    SAC architecture (same as DQN)
 
-=============  ==============  ==============  ==================
- Atari env     RLlib PPO @10M  RLlib PPO @25M  Baselines PPO @10M
-=============  ==============  ==============  ==================
-BeamRider      2807            4480            ~1800
-Breakout       104             201             ~250
-Qbert          11085           14247           ~14000
-SpaceInvaders  671             944             ~800
-=============  ==============  ==============  ==================
+RLlib's soft-actor critic implementation is ported from the `official SAC repo <https://github.com/rail-berkeley/softlearning>`__ to better integrate with RLlib APIs.
+Note that SAC has two fields to configure for custom models: ``policy_model_config`` and ``q_model_config``, the ``model`` field of the config is ignored.
 
+Tuned examples (continuous actions):
+`Pendulum-v1 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/sac/pendulum-sac.yaml>`__,
+`HalfCheetah-v3 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/sac/halfcheetah-sac.yaml>`__,
+Tuned examples (discrete actions):
```

Comment:
Discrete actions are not implemented in the new stack.

Reply:
True! Good catch.

Reply:
The table has this correctly.

Reply:
Ah alright. Must have overseen it there.